src/node_util.cc

```diff
@@ -254,12 +254,53 @@ static void ParseEnv(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(dotenv.ToObject(env));
 }
 
+static void GetCallSite(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  Isolate* isolate = env->isolate();
+
+  Local<StackTrace> stack =
+      StackTrace::CurrentStackTrace(isolate, env->stack_trace_limit());
```

Comment:
Should you mention in the docs the amount of callsite objects is limited by the `Error.stackTraceLimit`?

Reply:
Also, maybe this value should be customizable.

Reply:
It is not limited by Error.stackTraceLimit anymore. We use the environment const value (10).

Reply:
I think the request is to accept a parameter limiting the number of stack frames collected. For example, in the test runner, we call `StackTrace::CurrentStackTrace(2)` (in this same file) because we only care about the caller. Similarly, [module mocking uses the tamper-proof JS version of this API but doesn't need all of the stack frames](https://github.com/nodejs/node/blob/7c76fa0aee8e2aeedcfed5186cca98b6f429311b/lib/internal/test_runner/mock/mock.js#L511).

Reply:
Done in https://github.com/nodejs/node/pull/54380/commits/8d9f30ae07149427c85cfda0f27d8468aa465968